www/front_src/src/Resources/Listing/columns/Graph/index.tsx

```diff
@@ -57,7 +57,9 @@ const GraphColumn = ({
           <PerformanceGraph
             endpoint={endpoint}
             graphHeight={150}
+            resource={row}
             timeline={[]}
+            onAddComment={(): void => undefined}
```

Comment:
Why did you add a `onAddComment` property that implement any logic ?

Reply:
It's needed in case the timeline is displayed, so that the comment is added in real time instead of waiting for it to be retrieved by the API